src/EditorFeatures/Core/Host/IStreamingFindReferencesPresenter.cs

```diff
@@ -88,27 +87,27 @@ internal static class IStreamingFindUsagesPresenterExtensions
                 // If we're directly going to a location we need to activate the preview so
                 // that focus follows to the new cursor position. This behavior is expected
                 // because we are only going to navigate once successfully
-                if (await item.TryNavigateToAsync(workspace, new NavigationOptions(PreferProvisionalTab: true, ActivateTab: true), cancellationToken).ConfigureAwait(false))
-                    return true;
+                return new NavigableLocation(cancellationToken => item.TryNavigateToAsync(workspace, options, cancellationToken));
```

Comment:
I'm now confused why we introduced INavigableLocation when we also had an INavigableItem here. (I swore we had something earlier but couldn't remember what it was called...now I know. :smile:)

Reply:
So NavigableItem truly is "i have a single document and a span and i will go to that".  INavigableItem encapsulates the idea of just anything generic that can be navigated to.  You've seen already taht one example of that with the streaming-presenter 'navigating' by presenting a set of results.  Other examples include calling out to 3rd parties to take over the navigation (we call this '3rd party navigation) and take the user somewhere.